test/documentation/seqan3_footer.html

```diff
@@ -0,0 +1,23 @@
+<!-- HTML footer for doxygen 1.8.16-->
+<!-- start footer part -->
+<!--BEGIN GENERATE_TREEVIEW-->
+<div id="nav-path" class="navpath"><!-- id is needed for treeview function! -->
+  <ul>
+    <div id="list_bottom_right" style="text-align:right; float:left; color:black; padding: 0px 0px 0px 10px"></div>
+    <script type="text/javascript" charset="utf-8" src="../../js/version.js"></script>
```

Comment:
This is the actual non-standard doxygen stuff

Reply:
Where is version.js? This should be within the repository.

Reply:
Well, currently the setup (pre-existing) is like that you have the `version.js` which fills the dropdown menu and switches pages when you select something.
The possible choices are parsed by `version.php` which crawls trough the directory, collects the folder names and returns a json.

It would probably work if you ship both with the library if that's what you want.

Reply:
It's maybe kinda hard to tell the script where to look for the subfolders of the different versions...

Reply:
Yeah, please check all necessary files in :)

Reply:
Ok, done